llama-stack-client-kotlin-client-local/src/main/kotlin/com/llama/llamastack/client/local/util/ResponseUtil.kt

```diff
@@ -0,0 +1,61 @@
+package com.llama.llamastack.client.local.util
+
+import com.llama.llamastack.core.JsonValue
+import com.llama.llamastack.models.CompletionMessage
+import com.llama.llamastack.models.InferenceChatCompletionResponse
+import com.llama.llamastack.models.ToolCall
+import java.util.UUID
+
+fun buildInferenceChatCompletionResponse(
+    response: String,
+    stats: Float
+): InferenceChatCompletionResponse {
+    // check for prefix [ and suffix ] if so then tool call.
+    // parse for "toolName", "additionalProperties"
+
+    var completionMessage =
+        if (response.startsWith("[") && response.endsWith("]")) {
+            // custom tool call
+            CompletionMessage.builder()
+                .toolCalls(listOf(createCustomToolCall(response)))
```

Comment:
This list seems will only return one item in it. How is LS handling multiple tool calls right now?

Reply:
yes, that's true. I think the logic for how we want to carry this ultimately depends on the system prompt we have. I just saw our custom tool calling prompt says "[func_name1(params_name1=params_value1, params_name2=params_value2...), func_name2(params)]" so let me account for this with a for loop.

Reply:
added this in ec9e4bf43e58452786cf116f2e7f922c52947981